core/arch/arm/pta/sdp_pta.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2017, ARM Limited, All Rights Reserved
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Format
https://github.com/OP-TEE/optee_os/blob/master/documentation/copyright_and_license_headers.rst

Reply:
Ok, will change it to follow new format.